libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/model/eh/OpenLRStandard.kt

```diff
@@ -0,0 +1,11 @@
+package com.mapbox.navigation.core.trip.model.eh
+
+/**
+ * OpenLRStandard
+ */
+enum class OpenLRStandard {
```

Comment:
do we expect that this enum becomes wider? If so we should avoid using enum and replace by String+StringDef

Reply:
changed to StringDef